Automattic/jetpack
# Blaze: add new endpoints to access general site status info

> **Note**
> Let's not merge this until #30103 is merged, since this is added complexity that will make the original PR even more difficult to review.

## Proposed changes:

This will allow us to access general Blaze settings from other places in WP, from different plugins.

For example, this may come in handy in #31479.

### Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Start with a site that's connected to WordPress.com.
* Go to Posts > Add New, and open the JavaScript console.
* Type in `wp.apiFetch( { path: 'jetpack/v4/blaze/eligibility' } ).then( data => console.log( data ) );`
    * Your site should return `true` by default.
* Type in `wp.apiFetch( { path: 'jetpack/v4/blaze/dashboard' } ).then( data => console.log( data ) );`
    * Your site should return `false` by default.
* Now add the following to a functionality plugin:
```php
add_filter( 'jetpack_blaze_dashboard_enable', '__return_true' );
```
* Your site should now return `true` for the endpoint above.
